[on hold recsys work] Recommended For You

infinite-persistence opened the pull request:
## Remaining
<strike>- [ ] Fill in stub for "this is experimental" hover tooltip.</strike>
- [ ] Wider internal test audience.

## Ticket
Closes [#730 Add personalized recommendations to front page ](https://github.com/OdyseeTeam/odysee-frontend/issues/730)

## Implementation Notes
- The recsys code is temporarily placed in an non-ideal location until the `extras/` import issue is fixed.

infinite-persistence commented:
[62fd5d3 ](https://github.com/OdyseeTeam/odysee-frontend/commit/62fd5d375f74bf7c00f96652058be297fda3d1d3)to [e2cd6a3](https://github.com/OdyseeTeam/odysee-frontend/commit/e2cd6a36f0a4aa14c93bd76ee926facaaf1e8896)
- Rebased to latest master

tzarebczan commented:
- [ ] pass x-auth-token to all FYP requests: ```x-lbry-auth-token: ``` . Looks like there's a way to do with with beacon too...
- [ ] pass x-auth-token on recsys calls also

(johnny not sure if it will work yet, but he's on it)

infinite-persistence commented:
[e2cd6a3 ](https://github.com/OdyseeTeam/odysee-frontend/commit/e2cd6a36f0a4aa14c93bd76ee926facaaf1e8896)to [d71056d ](https://github.com/OdyseeTeam/odysee-frontend/commit/d71056d685740933e4410183e5ea960b845534c9)
- Pass auth-token to FYP calls, except for beacons (`mark`)
- Rebased to master; fixed css per new theme.

infinite-persistence commented:
[c02de5b ](https://github.com/OdyseeTeam/odysee-frontend/commit/c02de5b86706cb6a75054c5be8e7cf8fe263260c)to [9d0039e ](https://github.com/OdyseeTeam/odysee-frontend/commit/9d0039e1aae0e75941310e68f4848b9939203418)
- Point the "learn more" to `/$/fyp`.  
    - [ ] Change if necessary.
- Only show FYP section for members.
- Rebased

infinite-persistence commented:
[afcdbad ](https://github.com/OdyseeTeam/odysee-frontend/commit/afcdbad5dd35a063e6f6320f83471ffc6b1ead9e)to [946ba8d ](https://github.com/OdyseeTeam/odysee-frontend/commit/946ba8d208cca392f81db9adb8c9931259df5fed)
- `mark`: replace beacon with regular post
- Re-implement readme page using `MarkdownPreview`
    - Fixes mobile layout problem (damn css)
    - Experimenting with single i18n entry:  probably easier to translate as a whole.

tzarebczan commented:
Thank you! I'll work on the faq and try to get this out later today or tomorrow.

tzarebczan commented on ui/redux/actions/search.js:
```diff
@@ -85,31 +115,10 @@ export const doSearch = (rawQuery: string, searchOptions: SearchOptions) => (
+      const uris = processLighthouseResults(result);
 
+      const actions = [];
       actions.push(doResolveUris(uris));
```
we should try batching here, probably easier on the browser even if it's a bit slower (for FYP + related)
